.npmcheckrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this something specific of node 24?

Reply:
`vitest/coverage-v8` is from https://github.com/adobe/commerce-checkout-starter-kit/pull/54 (@babel/preset-react was already there, so same issue existed before for this), same pr as 24 upgrade but not related.

The test framework was migrated, and this start to generate noises in npx check, which is one of the steps we ask to check before app submission: https://developer.adobe.com/commerce/extensibility/app-development/app-submission-guidelines/#dependency-management

This file just snoozes warning of these two packages that npm wrongly considers the frameworks not in use. I thought it might be minor change so didn't include in the title.